foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -25,15 +25,15 @@ In other words, in our very first example, we put `display: flex` on a div and i
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-One thing to note is that this example would not work if we used the shorthand `flex: 1`. Try it out now (i.e. go change the flex value on line 11). Can you figure out why? The divs collapse, even though they _clearly_ have a `height` defined there.
+One thing to note is that this example would not work if we used the shorthand `flex: 1`. Try it out now (i.e. go change the flex value on the `flex: 1 1 auto;` line). Can you figure out why it does not work if `flex: 1` is used? The divs collapse, even though they _clearly_ have a `height` defined there.
```

Comment:
There are two reasons I suggested the change shown, both of which have to do with how the code referenced is displayed on the lesson page embedding (not the codepen web site):

1) In the lesson page embedding there are no line numbers shown, so it's a pain for the user to count them.

2) Even if the user does count the lines, the referenced line doesn't actually appear as line 11 in the embedding. Depending on how the window is sized, the commented line that starts with *this selector selects* varies in how many lines it takes up, and while that doesn't matter on the codepen web site because the line numbers are just spaced out, without numbers for reference in the lesson page embedding it is easy to include those lines in a line count and arrive at a value of 12, 13 or 14 for the desired line.

That was my experience using the page, and why I suggested this change. Another fix option would be to point out that it's only clearly marked line 11 on the codepen web site.

Thanks!